docs/UserGuide.md

```diff
@@ -130,10 +169,11 @@ In the current workout list, there are a total of 3 workouts. All the workouts h
 If there are 20 workouts in the list, all the 20 workouts will be displayed together. 
 
 ---
+
 ### Show specific number of Workouts: `workout /list`
 Lists down a specific number of workouts that have been created and stored in the workout list at a time.
 The maximum number of workouts that will be displayed at a time is 10 workouts. If there are more
-workouts in the list, user will be prompt for continuation of printing.
+workouts in the list, you will be prompt to give permission to continue the printing.
```

Comment:
Perhaps the phrase "..., you will be prompt to give permission to continue the printing" can be reworded to
> ", ... you will be **prompted** on **whether you wish to display the next 10 (or fewer) workouts in the list**."

It's wordier, but I feel it has a softer tone for the reader. 🙂

Reply:
Yes, i agree with you